.nvmrc

```diff
@@ -1 +1 @@
-v20.11.1
+v20
```

Comment:
Just to be safe, might be better to set this to the same exact version as the workflows

Reply:
Do you want to freeze `patch` versions, too?

Reply:
I think so, so as to keep it consistent. Then a single search for the version number will also reveal all the places in which it has to be updated.